chart/compass/charts/prometheus-postgres-exporter/values.yaml

```diff
@@ -295,40 +295,40 @@ config:
       master: true
       cache_seconds: 43200
       metrics:
         - quantity:
             usage: "COUNTER"
-            description: "Number of TRIAL tenants that have at least one non auto-discovered Application (the tenant is owner and its parent is null) or tenants that have at least 1 formation different from the default one where their parent is either null or their parent is of type customer"
+            description: "Number of TRIAL tenants that have at least one non auto-discovered Application (the tenant is owner and its parent is null) or tenants that have at least on system in state different from 'INACTIVE' or tenants that have at least 1 formation different from the default one where their parent is either null or their parent is of type customer"
```

Comment:
for all three descriptions - `on system in state` -> `one system in state`, `1 formation` -> `one formation`.